canonical-web-and-design/dqlite.io
# Update to Vanilla 2.12

## Done

- Updates to latest Vanilla 2.12.0
- Removes local overrides for side navigation and docs layout styles

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8037
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to [/docs](https://dqlite-io-canonical-web-and-design-pr-52.run.demo.haus/docs), make sure side navigation works properly on large and small screens
